scala/scala-impl/src/org/jetbrains/plugins/hydra/HydraRepository.scala

```diff
@@ -0,0 +1,24 @@
+package org.jetbrains.plugins.hydra
+
+import org.jetbrains.plugins.hydra.settings.HydraApplicationSettings
+
+/**
+  * @author Maris Alexandru
+  */
+object HydraRepository {
+
+  private val globalSettings = HydraApplicationSettings.getInstance()
```

Comment:
This field seems useless. Just inline the value where it's used (if you did it for performance reasons, the JVM is usually good at optimizing this kind of calls)